src/admin_upload/views.py

```diff
@@ -0,0 +1,89 @@
+from django.shortcuts import render, redirect
+from django.http import HttpResponse, JsonResponse
+from django.contrib.auth.decorators import user_passes_test
+from geonode.layers.models import Dataset
+import json
+import psycopg2
+import sys
+from woolpert import settings
+
+def admin_required(function):
+    def wrap(request, *args, **kwargs):
+        user = request.user
+        if user.is_superuser:
+            return function(request, *args, *kwargs)
+        else:
+            return redirect("/")
+    wrap.__doc__ = function.__doc__
+    wrap.__name__ = function.__name__
+    return wrap
+
+# Create your views here.
+@admin_required
+def admin_form(request):
+    if(request.POST):
+        json_data = json.loads(request.POST.get('json_data'))
+        x_index = [i for i, x in enumerate(json_data["columns"]) if x == 'x_dd']
+        y_index = [i for i, x in enumerate(json_data["columns"]) if x == 'y_dd']
+
+        dbHost     = settings.DATABASE_HOST
```

Comment:
These values might change, we might not be getting them from `settings.` but should be fetched from env for a separate DB which we will have by Monday

Reply:
@NyakudyaA  The values that are being set here is fetched from settings.py that uses an env variable

Reply:
Yes, But I just meant we are going to have a separate database from the GeoNode DB, which could come from a maybe a domain like `woolpertdb.com`, so the GeoNode would not know about those credentials. So perhaps we have a page in the admin page to allow a user to enter the credentials `DB_HOST,DB_USER,DB_PASS,DB_PORT`